### DISCUSSION TITLE
Are spelling mistakes kept on purpose?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#2381] [General]
Codespell got added to the readme, and its icon is always red, indicating some mistakes to be found. When [viewing](https://github.com/sandboxie-plus/Sandboxie/actions/workflows/codespell.yml) the codespell actions page, it does not appear that those mistakes are sorted out. Is there some rationale behind that?

**Comment by DavidXanatos** [comment]
> Is there some rationale behind that?

laziness and/or lack of spare time...

fill free to submit a pull request resolving those

**Reply by bastik-1001** [reply]
I might do that, if I can be certain that proposed changes will be correct.
